windows-utils/replace-symlinks.js

```diff
@@ -0,0 +1,94 @@
+const shell = require("shelljs");
+
+async function getSymlinks(dirPath) {
+    let symlinks = [];
+    const path = dirPath
+        .replaceAll("/", "\\")
+        .replaceAll("../", "")
+        .split("\\")
+        .slice(2)
+        .join("\\");
+    const cwd = process.cwd();
+    const changeDir = cwd + path;
+    const fileList = await shellExec("git ls-files -s");
+
+    if (fileList.stdout.includes("120000")) {
```

Comment:
So, this made me stumble :D Turns out it's showing file type and permission ("mode") in ancient Unix terminology that's so ancient that I don't remember ever seeing it.

Maybe the next person reading this can safe some time if we put it into a constant like `symlinkMode`. The "official" constant for this would be called `S_IFLNK` from what I found: https://github.com/torvalds/linux/blob/5bfc75d92efd494db37f5c4c173d3639d4772966/include/uapi/linux/stat.h#L11 - But that's a bit cryptic :D